### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#775] [Galaxy Trivia] [answered]
> *Do as I ... not as I ...*

**Bonus:** Name the author

**Comment by peterbaccaro** [comment, marked as answer]
The expression **Do as I say, not as I do**, can be traced to **John Selden**’s work, Table-Talk, published in 1654: “Preachers say, ‘Do as I say, not as I do.’”

The origins of the expression however are uncertain, and it's not clear who first coined the phrase. However, it's a common saying that has been used by many people over time, including parents, teachers, and leaders. The phrase has become a kind of proverb, a short, wise saying that expresses a universal truth or moral lesson. It emphasizes the importance of following good advice, even if the person giving the advice does not always set a perfect example.

There is no evidence to suggest that John Selden came up with the proverb "Do as I say, not as I do." The exact origins of this phrase are uncertain, but it is believed to have been in use for several centuries before Selden's time.

**Comment by SANKARAMDAS** [comment]
Do As I Say, Not As I Do
The origin of the phrase “do as I say, not as I do” actually came from Bible) from the book of Matthew, verses 1 to 3. The verses state the same meaning of this phrase, just in different words. Since then, different wording has been used to present this same principle throughout history. The first appearance of the actual phrasing “do as I say, not as I do” can be found in the book Table Talk written by author John Selden in 1654, published after his death in 1689.
